src/App.js

```diff
@@ -1,5 +1,147 @@
-function App() {
-	return <div className='App'>App</div>
-}
+import axios from 'axios'
```

Comment:
UI suggestion: The UI looks really great the suggestion I have is that the images of the background are dynamic and it can be anything that the user selects. so with some images, the text will be hard to read so you can do this little trick to make it more readable 
> ![image](https://user-images.githubusercontent.com/67889207/165152225-83a7ecea-27f0-4612-9096-37e86271f1a8.png)

check it out and let me know your opinion

Reply:
This is great